### PR TITLE
chore(docs): slider and range slider docs update (VIV-2105)

### PR DESCRIPTION
**Pull request by rachelbt**

### CODE REVIEW

File: libs/components/src/lib/slider/README.md [hunk 1]
```diff
@@ -60,12 +98,10 @@ Use `max` to set the greatest value allowed for the slider.
...
 
-### Step
+## Step
 
-Use `step` to set determine the granularity with which values can be incremented/decremented.
-
-- Type: `number`
-- Default: `1`
+Use the `step` attribute sets the granularity with which values can be incremented/decremented.  
+The default value `step` of is `1`.
```
Comment (TaylorJ76) [inline libs/components/src/lib/slider/README.md:104]:
```suggestion
The default value of `step` is `1`.
```

**Review by TaylorJ76 — CHANGES_REQUESTED** [review]

File: libs/components/src/lib/range-slider/VARIATIONS.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+## Connotation
+
+Use the `connotation` attribute on Range Slider to control the color of the selected range.  
+Below is an example of cta connotation.
+
+```html preview blocks>
+<vwc-range-slider connotation="cta"></vwc-range-slider>
```
Comment (TaylorJ76) [inline libs/components/src/lib/range-slider/VARIATIONS.md:7]: Can we show both connotations?

File: libs/components/src/lib/range-slider/VARIATIONS.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+```
+
+## Markers
+
+Use `markers` attribute to add tick marks on the slider.
```
Comment (TaylorJ76) [inline libs/components/src/lib/range-slider/VARIATIONS.md:12]:
```suggestion
Use the `markers` attribute to add tick marks on the slider.  
```

File: libs/components/src/lib/slider/VARIATIONS.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+Use the `connotation` attribute on Slider to control the Slider track.  
+Below is an example of cta connotation.
+
+```html preview blocks>
+<vwc-slider connotation="cta"></vwc-slider>
+```
+
+## Markers
+
+Use `markers` attribute to add tick marks on the slider.
```
Comment (TaylorJ76) [inline libs/components/src/lib/slider/VARIATIONS.md:12]:
```suggestion
Use the `markers` attribute to add tick marks on the slider.  
```

File: libs/components/src/lib/range-slider/README.md [hunk 5]
```diff
@@ -45,16 +72,14 @@ The lowest value allowed for the range.
...
 
-### Max
+## Max
 
-The highest value allowed for the range.
-
-- Type: `number`
-- Default: `0`
+Use the `max` attribute to set the highest value allowed for the Range Slider.  
+The default value `max` of is `0`.
```
Comment (TaylorJ76) [inline libs/components/src/lib/range-slider/README.md:78]:
```suggestion
The default value `max` of is `10`.
```

**Review by TaylorJ76 — CHANGES_REQUESTED** [review]

File: libs/components/src/lib/slider/README.md [hunk 6]
```diff
@@ -84,78 +120,27 @@ Use `step` to set determine the granularity with which values can be incremented
...
-
-Use `value` to set the value of the slider.
+<div class="table-wrapper">
 
-- Type: `string`
-- Default: `5`
+| Name            | Type                                    | Description                                                          |
+| --------------- | --------------------------------------- | -------------------------------------------------------------------- |
+| **connotation** | _Enum_:<br/>`accent`<br/>`cta`          | The connotation of the component                                     |
+| **disabled**    | \_`boolean`                             | Sets the element's disabled state                                    |
```
Comment (rachelbt) [inline libs/components/src/lib/slider/README.md:132]:
```suggestion
| **disabled**    | `boolean`                             | Sets the element's disabled state                                    |
```

**Review by TaylorJ76 — APPROVED** [review]

**Pull request merged**